src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/validation/BotusValidationService.kt

```diff
@@ -1,33 +1,47 @@
 package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.validation
 
-import org.slf4j.LoggerFactory
 import org.springframework.stereotype.Service
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.config.FeatureToggles
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.SentenceAndOffenceWithReleaseArrangements
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.external.PrisonApiSourceData
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.external.SentenceAndOffence
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.external.SentenceCalculationType
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.external.SentenceCalculationType.BOTUS
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.validation.ValidationCode.BOTUS_CONSECUTIVE_OR_CONCURRENT_TO_OTHER_SENTENCE
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.validation.ValidationCode.BOTUS_CONSECUTIVE_TO_OTHER_SENTENCE
 
 @Service
 class BotusValidationService(
   private val featureToggles: FeatureToggles,
 ) {
 
   internal fun validate(sourceData: PrisonApiSourceData): List<ValidationMessage> {
-    if (featureToggles.botusConsecutiveJourney) {
-      log.info("BOTUS consecutive and concurrent journeys are feature enabled - therefore the 'unsupported' validation for BOTUS will not run")
-      return emptyList()
+    if (featureToggles.botusConcurrentJourney) {
+      val consecutiveSentences = sourceData.sentenceAndOffences
+        .filter { it.consecutiveToSequence != null }
+
```

Comment:
not sure if I'm right on this but I thought both feature toggles could be on at the same time; I may be wrong though, worth checking with Katie

With this new code it looks like if CONCURRENT is on then it will always fail the consec check even if the CONSEC feature toggle is on